app/services/ip_info_service.rb

```diff
@@ -0,0 +1,7 @@
+class IpInfoService
+  def self.get_timezone
+    response = Faraday.get("https://ipinfo.io/json")
+    body = JSON.parse(response.body, symbolize_names: true)
+    body[:timezone]
```

Comment:
This won't work, because it'll get the time zone of the *server*, not the client.

Can we not have the user just send their local timezone in the form? JavaScript will know what it is via `getTimezoneOffset`.

Reply:
@dorner I've been attempting to get this to work but I've found that JavaScript isn't supported or not recommended (correct me if I'm wrong).

I have two potential solutions: 
1. Add the user's timezone at sign up so it's stored to the db. 
2. Mention that it will expire 8 hours.

I looked back at a lot of password emails I've gotten and it looks like they all just mention that it will expire in X hours. Would you suggest we do that as well?

Reply:
I think saying "it will expire in 8 hours" should be fine. @cielf ?

Reply:
Yup